storage/src/freezer/storage.rs

```diff
@@ -787,9 +787,6 @@ impl<E: Storage + Metrics + Clock, K: Array, V: Codec> Freezer<E, K, V> {
     /// Get the value for a given [Cursor].
     async fn get_cursor(&self, cursor: Cursor) -> Result<Option<V>, Error> {
```

Comment:
This can now also be `Result<V, Error>`?

Reply:
yup, fixed